File: Modul 0/Soal 4/PRAK004-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>
     
int main() {
    char tagar[] = "#                            #";
    int i;
    char namaLengkap[] = "Muhammad Bukhari Fitri";
    long long int nim = 2310817210015LL;
    int barisAwal = 1;
    int jmlBaris = 6;
    for(i = barisAwal; i <= jmlBaris; ++i) {
        
        
        if ( i == jmlBaris/2 ) {
            printf("#   %s   #", namaLengkap);
        }

        else if ( i == jmlBaris/2+1 ) {
             printf("#       %lld        #", nim);
        }
        else if (i == barisAwal || i == jmlBaris) {
            printf("##############################");
        }
        else{
            printf("%s", tagar);
        }
      printf("\n");
    }
}

File: Modul 0/Soal 4/PRAK004-2310817210015-MuhammadBukhariFitri.py
tagar = "#                            #"
namaLengkap = "Muhammad Bukhari Fitri"
nim = "2310817210015"
barisAwal = 1
jmlBaris = 6

for i in range(1, jmlBaris+1):
    if i == jmlBaris/2:
        print("#  ", namaLengkap, "  #")
    elif i == jmlBaris/2+1:
        print("#      ", nim, "       #")
    elif i == barisAwal or i == jmlBaris:
        print("##############################")
    else:
        print(tagar)
